crates/accelerate/src/commutation_checker.rs

```diff
@@ -652,13 +605,19 @@ fn map_rotation<'a>(
 ) -> (&'a OperationRef<'a>, &'a [Param], bool) {
     let name = op.name();
 
-    if let Some((pi_multiple, generator)) = SUPPORTED_ROTATIONS.get(name) {
+    if let Some(generator) = SUPPORTED_ROTATIONS.get(name) {
```

Comment:
Maybe not for this PR but we can drop the hashset here all together and have a static lookup table based on the standard gates. Something like:

```rust
const fn builld_lut() -> [Option<StandardGate>; STANDARD_GATE_SIZE]  {
    ...
}
let static supported_rotations: [Option<StandardGate>; STANDARD_GATE_SIZE] = build_lut()
```

Alternatively you could just do a couple of if `matches!()` on `.standard_gate()`. 

When in rust space and working solely with standard gates we really never have a need for strings, so this just sticks out to me every time I see it. But it was pre-existing so we can do this in a follow up.

Reply:
Yeah that sounds like a good idea, I'll put it on the follow-up list if that's good for you 🙂